packages/types/types/test/record.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import "mocha";
+import { strict as assert } from "assert";
+import { IReader, IWriter } from "../src";
+
+describe("Record", () => {
```

Comment:
Sanity check that our record interfaces are compatible w/ES6 map...  any other tests?